komga-webui/src/locales/en.json

```diff
@@ -23,7 +23,7 @@
   },
   "authentication_activity": {
     "datetime": "Date Time",
-    "email": "Email",
```

Comment:
This is strictly speaking the less traditional form, and it also produces errors in translation (where this isn't allowed).

Reply:
Can you expand on the errors in translation? Google uses `email` on their own websites for instance.

Reply:
They also have all professional translators, even though results vary.
If the source says `a-b`, people generally remember to use `-` in their language when it is (often the case) required, whereas if it is `ab`, it is invariably missed.